packages/cli/src/ui/hooks/useGeminiStream.ts

```diff
@@ -501,6 +530,12 @@ export const useGeminiStream = (
           case ServerGeminiEventType.MaxSessionTurns:
             handleMaxSessionTurnsEvent();
             break;
+          case ServerGeminiEventType.Finished:
+            handleFinishedEvent(event as ServerGeminiFinishedEvent, userMessageTimestamp);
+            break;
+          case ServerGeminiEventType.UsageMetadata:
```

Comment:
Hmm, lets drop this if we could from the PR. Would like to keep this more targeted for now